<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12861" started-at="2020-12-15T10:24:01Z" finished-at="2020-12-15T10:24:13Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12861" started-at="2020-12-15T10:24:01Z" finished-at="2020-12-15T10:24:13Z">
      <class name="com.learnautomation.framework.testcases.LoginTest">
        <test-method status="PASS" signature="startBrowser()[pri:0, instance:com.learnautomation.framework.testcases.LoginTest@69ea3742]" name="startBrowser" is-config="true" duration-ms="9129" started-at="2020-12-15T15:54:01Z" finished-at="2020-12-15T15:54:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startBrowser -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.learnautomation.framework.testcases.LoginTest@69ea3742]" name="setup" is-config="true" duration-ms="14" started-at="2020-12-15T15:54:12Z" finished-at="2020-12-15T15:54:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginWithAdmin(java.lang.String, java.lang.String)[pri:0, instance:com.learnautomation.framework.testcases.LoginTest@69ea3742]" name="loginWithAdmin" duration-ms="1521" started-at="2020-12-15T15:54:12Z" data-provider="SingleLogin" finished-at="2020-12-15T15:54:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.learnautomation.framework.pages.LoginPage.loginToApplication(LoginPage.java:46)
	at com.learnautomation.framework.testcases.LoginTest.loginWithAdmin(LoginTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithAdmin -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.learnautomation.framework.testcases.LoginTest@69ea3742]" name="closeBrowser" is-config="true" duration-ms="210" started-at="2020-12-15T15:54:13Z" finished-at="2020-12-15T15:54:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.learnautomation.framework.testcases.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
